test/services/VatDetailsServiceSpec.scala

```diff
@@ -31,8 +31,7 @@ import play.api.test.Helpers.{await, defaultAwaitTimeout}
 import scala.concurrent.{ExecutionContext, Future}
 
 class VatDetailsServiceSpec extends ControllerBaseSpec {
-
-  private trait Test {
+  
```

Comment:
As mentioned on Slack, if you could remove most of the payment models from this spec to reduce bloat as only one is needed to prove the test